src/pages/en/install/manual.md

```diff
@@ -146,7 +146,24 @@ If you want to include [UI framework components](/en/core-concepts/framework-com
 
 📚 Read Astro's [API configuration reference](/en/reference/configuration-reference/) for more information.
 
-## 6. Next Steps
+## 6. Create `tsconfig.json`
+
+Typescript is configured using `tsconfig.json`. Even if you don’t write TypeScript code, this file is important so that tools like Astro and VS Code know how to understand your project. Some features (like npm package imports) aren’t fully supported in the editor without a `tsconfig.json` file.
```

Comment:
```suggestion
Typescript is configured using `tsconfig.json`. Even if you don’t write TypeScript code, this file using Astro's `base` template is important so that tools like Astro and VS Code know how to understand your project. Some features (like npm package imports) aren’t fully supported in the editor without a `tsconfig.json` file. If you intend to write TypeScript code, using Astro's `strict` or `strictest` template is recommended. 
```